scripts/run_lighthouse_tests.py

```
@@ -169,52 +156,142 @@ def get_entity(line: str) -> tuple[str, str] | None:
+    the donate page). Without this patch, Chrome's headless CDP rejects
+    Target.getTargetInfo for cross-origin targets and Lighthouse crashes.
     """
+    target_manager_path = os.path.join(
+        'node_modules',
+        'lighthouse',
+        'core',
+        'gather',
+        'driver',
+        'target-manager.js',
+    )
+    with open(target_manager_path, 'r', encoding='utf-8') as f:
```

Ruff (0.16.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)